src/ResourceManager/LocationBasedServices/Commands.LocationBasedServices/LocationBasedServicesAccount/NewAzureLocationBasedServicesAccount.cs

```diff
@@ -0,0 +1,110 @@
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Management.LocationBasedServices;
+using Microsoft.Azure.Management.LocationBasedServices.Models;
+using Microsoft.Azure.Commands.LocationBasedServices.Properties;
+using System.Collections;
+using System.Globalization;
+using System.Management.Automation;
+using LocationBasedServicesModels = Microsoft.Azure.Commands.LocationBasedServices.Models;
+
+namespace Microsoft.Azure.Commands.LocationBasedServices
+{
+    /// <summary>
+    /// Create a new Location Based Services Account, specify it's type (resource kind)
+    /// </summary>
+    [Cmdlet(VerbsCommon.New, LocationBasedServicesAccountNounStr, SupportsShouldProcess = true), OutputType(typeof(LocationBasedServicesModels.PSLocationBasedServicesAccount))]
+    public class NewAzureLocationBasedServicesAccountCommand : LocationBasedServicesAccountBaseCmdlet
+    {
+        [Parameter(
+            Position = 0,
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Resource Group Name.")]
+        [ValidateNotNullOrEmpty]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(
+            Position = 1,
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Location Based Services Account Name.")]
+        [Alias(LocationBasedServicesAccountNameAlias, AccountNameAlias)]
+        [ValidateNotNullOrEmpty]
+        [ValidatePattern("^[a-zA-Z0-9][a-zA-Z0-9_.-]*$")]
+        [ValidateLength(2, 64)]
+        public string Name { get; set; }
+
+        [Parameter(
+            Position = 2,
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Location Based Services Account Sku Name.")]
+        [ValidateSet("S0")]
+        public string SkuName { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Location Based Services Account Tags.")]
+        [Alias(TagsAlias)]
+        [ValidateNotNull]
+        [AllowEmptyCollection]
+        public Hashtable[] Tag { get; set; }
+
+        [Parameter(Mandatory = false, HelpMessage = "Don't ask for confirmation.")]
+        public SwitchParameter Force { get; set; }
+
```

Comment:
Must implement AsJob. See: https://github.com/Azure/azure-powershell/wiki/PowerShell-Cmdlet-Design-Guidelines#asjob-parameter

Reply:
Creation for this resource is not a long-running operation; it is immediate.